src/item/components/modals/AddToCollectionModal.tsx

```diff
@@ -76,7 +76,17 @@ const AddToCollectionModal: FunctionComponent<AddToCollectionModalProps & UserPr
 				[],
 				ContentTypeNumber.collection,
 				undefined,
-				undefined
+				undefined,
+				[
+					{
+						_not: {
+							contributors: {
+								profile_id: { _eq: getProfileId(user) },
+								rights: { _neq: 'CONTRIBUTOR' },
+							},
+						},
+					},
+				]
```

Comment:
deze query vind ik nogal moeilijk te interpreteren door de dubbele negatie
ik heb het ff getest met jouw oplossing voor lesgever secundair met de 3 soorten collecties:
* eigenaar
* bewerker
* bekijker

![image](https://github.com/viaacode/avo2-client/assets/1710840/4e7553a3-a05d-407a-bafa-9b4fbe7e67a5)

De resultaten voor je query zijn inderdaad correct:
![image](https://github.com/viaacode/avo2-client/assets/1710840/b2604520-7b50-4e38-b784-f55583d3cc2a)


Deze query lijkt me wel logisch en geeft dezelfde resultaten. Je moet ofwel bewerker zijn ofwel de eigenaar
```suggestion
				[
					{
						_or: [
							{
								contributors: {
									rights: { _eq: 'CONTRIBUTOR' },
								},
							},
							{
								owner_profile_id: { _eq: getProfileId(user) },
							},
						],
					},
				]
```

resultaat:
![image](https://github.com/viaacode/avo2-client/assets/1710840/a694acf6-6ce1-40e0-a6d2-94bf87c83a00)